tuf/ngclient/updater.py

```diff
@@ -0,0 +1,642 @@
+# Copyright 2020, New York University and the TUF contributors
+# SPDX-License-Identifier: MIT OR Apache-2.0
+
+"""TUF module that implements the client update workflow.
+
+This module contains the Updater class that provides an implementation of the
+TUF client workflow specified in
+https://theupdateframework.github.io/specification/latest/#detailed-client-workflow.
```

Comment:
If we are going for the linking, this could become:

```suggestion
 `TUF client workflow <https://theupdateframework.github.io/specification/latest/#detailed-client-workflow>`_ 
```